alphagov/paas-cf
# [#157190168] Cleanup manifests following cf-deployment migration (part 3)

What
----

This is the final part of the manifest cleanup following the cf-deployment migration.

How to review
-------------

* Deploy #1354 (or master if that's been merged)
* Rotate the leaf certificates (Rotate the leaf certificates (step 3+4 [here](https://team-manual.cloud.service.gov.uk/team/rotating_credentials/#rotating-certificates)), but **not the CA certs** (rotating CA certs is currently know to cause downtime)
* Deploy from this branch
* Verify all works
* Rotate the leaf certificates again
* Verify all works

:rotating_light: **Before merging** #1354 must be merged and deployed, and the leaf certificates rotated in all persistent environments before this is merged :rotating_light: 

Who can review
--------------

Not me.
